sklearn/linear_model/least_angle.py

```diff
@@ -196,7 +196,7 @@ def lars_path(X, y, Xy=None, Gram=None, max_iter=500,
 
     if verbose:
         if verbose > 1:
-            print("Step\t\tAdded\t\tDropped\t\tActive set size\t\tC")
+            print('Step\t\tAdded\t\tDropped\t\tActive set size\t\tC')
```

Comment:
On what basis is this a worthwhile change?

Please don't waste reviewers' time on conforming to a personal preference.

Such changes may also introduce merge conflicts to other substantive pull requests.